### ISSUE TITLE
html report

### ISSUE THREAD

**Issue by LecrisUT** [rpm-software-management/rpmlint#1156]
Something like how `rpminspect` bundles a `viewer.html` [^1] file that converts a json result to a simple web view would be nice.

For context, there is a `packit`/`tmt` plan [^2] that packagers can use to easily call `rpmlint`, and we've replicated the `rpminspect` viewer over there, and it would be nice if we can provide something similar for `rpmlint`.

[^1]: https://github.com/rpminspect/rpminspect/blob/main/contrib/viewer.html
[^2]: https://github.com/packit/tmt-plans/tree/main/plans/rpmlint

**Comment by danigm** [comment]
Right now rpmlint doesn't have any output format configuration, the output is done printing directly the errors. A first step could be to add some kind of configuration to output the information in a structured way, for example `json`, and then we can extend that option to support different formats, `json`, `xml`, `html`

**Comment by LecrisUT** [comment]
Indeed, on the backend `rpminspect` simply exports to `json` and the `viewer.html` simply reads it directly.
